### PR TITLE
Adding onDieCache related properties to .env files

### PR DESCRIPTION
**Pull request by DukeDavis12**
Adding onDieCache related properties to .env files.

Signed-off-by: Davis Benny <davis.benny@intel.com>

### CODE REVIEW

File: component-samples/demo/manufacturer/manufacturer.env [hunk 1]
```diff
@@ -6,6 +6,15 @@ manufacturer_database_port=8049
 catalina_home=./target/tomcat
 manufacturer_api_user=apiUser
 
+## PRI Manufacturer onDie Congfigurations
+## Uncomment the following lines to enable onDie support
+
+#ondie_cache=file:///home/manufacturer/ondie_cache/
```
Comment (trbehera) [inline component-samples/demo/manufacturer/manufacturer.env:12]: The user name is 'fdo' for all components. 
ondie_cache=file:///home/fdo/ondie_cache/

File: component-samples/demo/owner/Dockerfile [hunk 2]
```diff
@@ -10,6 +10,9 @@ RUN useradd -ms /bin/bash fdo
 WORKDIR /home/fdo/
 RUN mkdir -p /home/fdo/lib /home/fdo/serviceinfo
 
+# Uncomment the following line to enable onDie support for Owner
+# COPY --chown=owner ./ondie_cache ./ondie_cache/
```
Comment (trbehera) [inline component-samples/demo/owner/Dockerfile:14]: chown is performed in line 24 for all files, not required here.

File: component-samples/demo/owner/owner.env [hunk 3]
```diff
@@ -14,6 +14,14 @@ owner_svi_values=./serviceinfo/sample-values
 owner_svi_string=./serviceinfo/sample-svi.csv
 owner_api_user=apiUser
 
+## PRI Owner onDie Congfigurations
+## Uncomment the following lines to enable onDie support
+
+#ondie_cache=file:///home/owner/ondie_cache/
```
Comment (trbehera) [inline component-samples/demo/owner/owner.env:20]: file:///home/fdo/ondie_cache/

File: component-samples/demo/manufacturer/manufacturer.env (hunk 1 above)
Comment (DukeDavis12) [reply component-samples/demo/manufacturer/manufacturer.env]: Done.

File: component-samples/demo/owner/owner.env (hunk 3 above)
Comment (DukeDavis12) [reply component-samples/demo/owner/owner.env]: Done.

File: component-samples/demo/owner/Dockerfile (hunk 2 above)
Comment (DukeDavis12) [reply component-samples/demo/owner/Dockerfile]: Done.

**Review by trbehera — APPROVED** [review]

**Review by KiranSukhavasi — APPROVED** [review]

**Pull request merged**